pin/generate_pin

```diff
@@ -51,7 +51,7 @@ do
 done
 shift $((OPTIND -1))
 
-GITREF=${1:-'nixos-unstable'}
+GITREF=${1:-'master'}
```

Comment:
should this point to `master` or `release-21.05`?

Reply:
Good question! My thought process was: `master` as the default here since it was previously `unstable` when we were point to the old nix channel repo (RIP). Plus we wont have to find and replace this every time we jump to a new release.

You can then just pass a release to `generate_pin 21.05`